InteractionBase/src/vis/CommandPrompt.cpp

```diff
@@ -143,7 +145,7 @@ void CommandPrompt::determineChildren()
 {
 	GenericHandler* h = dynamic_cast<GenericHandler*> (handler());
 
-	if (h)
+	if (h && !hideRequested_)
```

Comment:
Essentially, if I use my CRemoveNodeFromView command to remove a node, Envision crashes. The issue seems to be that this function is called after the command is executed, and after the grid has already updates its contents (and therefore deleted the command receiver item).
With this, I can ensure the suggestions are not reloaded for a non-existing item.